### PR TITLE
Add deephunt3r to the list

### PR DESCRIPTION
**Pull request by deephunt3r**
Added my profile 😃 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/335"></a>

### CODE REVIEW

File: public/data/Deephunt3r.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Rakesh vaddi",
```
Comment (CodingSpecies) [inline public/data/Deephunt3r.json:2]: Should `vadi` be capitalised to `Vaddi`? Thank you.

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

File: public/data/Deephunt3r.json (hunk 1 above)
Comment (deephunt3r) [reply public/data/Deephunt3r.json]: @CodingSpecies Thanks for knowing me 👍🏻

**Review by CodingSpecies — APPROVED** [review]
LGTM! Thank you @deephunt3r

**Review by aayush89890 — COMMENTED** [review]
Looks good, Welcome to the community @deephunt3r .

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Comment by deephunt3r** [comment]
Thanks everyone for reviewing the pr 🙂

**Pull request merged**